region/maridia/inner-pink/Crab Shaft.json

```diff
@@ -1212,68 +1105,132 @@
       "devNote": "Requires entering on either side of the doorway, but not in the center."
     },
     {
-      "id": 62,
+      "id": 63,
       "link": [3, 3],
-      "name": "Carry G-Mode Back Up",
+      "name": "Carry G-Mode Morph Back Up",
       "entranceCondition": {
         "comeInWithGMode": {
           "mode": "direct",
-          "morphed": false
+          "morphed": true
         },
         "comesThroughToilet": "any"
       },
       "requires": [
         {"or": [
-          "Gravity",
-          "HiJump",
-          "canSpringBallJumpMidAir",
           {"and": [
-            "canTrickyUseFrozenEnemies",
-            {"ammo": {"type": "Super", "count": 1}}
+            "Gravity",
+            "h_canArtificialMorphIBJ"
+          ]},
+          {"and": [
+            "Gravity",
+            "h_canArtificialMorphSpringBall"
+          ]},
+          {"and": [
+            "HiJump",
+            "h_canArtificialMorphSpringBall"
           ]}
         ]}
       ],
       "exitCondition": {
         "leaveWithGMode": {
-          "morphed": false
+          "morphed": true
         }
       },
       "bypassesDoorShell": true,
       "flashSuitChecked": true
     },
     {
-      "id": 63,
-      "link": [3, 3],
-      "name": "Carry G-Mode Morph Back Up",
+      "link": [3, 4],
+      "name": "Fall Into Room",
       "entranceCondition": {
-        "comeInWithGMode": {
-          "mode": "direct",
-          "morphed": true
-        },
+        "comeInNormally": {},
         "comesThroughToilet": "any"
       },
+      "requires": []
+    },
+    {
+      "id": 58,
+      "link": [4, 1],
+      "name": "Base",
+      "requires": []
+    },
+    {
+      "link": [4, 3],
+      "name": "Base",
       "requires": [
         {"or": [
+          "Gravity",
+          "HiJump",
+          "canSpringBallJumpMidAir",
           {"and": [
-            "Gravity",
-            "h_canArtificialMorphIBJ"
-          ]},
+            "canTrickyUseFrozenEnemies",
+            {"ammo": {"type": "Super", "count": 1}}
+          ]}
+        ]}
+      ],
+      "flashSuitChecked": true
+    },
+    {
+      "link": [4, 3],
+      "name": "Crouch Jump",
+      "requires": [
+        {"or": [
+          "canCrouchJump",
+          "h_UnderwaterCrouchJumpWithFlashSuit"
+        ]}
+      ],
+      "flashSuitChecked": true
+    },
+    {
+      "id": 28,
+      "link": [4, 3],
+      "name": "Crab Ice Clip Door Lock Skip",
+      "requires": [
+        {"notable": "Crab Ice Clip Door Lock Skip"},
```

Comment:
Odd that these requirements didn't reference the tedious crab climb.
Starting from 4, the logic steps would tell you how to get up here before doing the crab clip.  And it would only be wrong if it wanted you to use SpeedBooster to climb.